blaze-core/src/main/scala/org/http4s/blazecore/util/IdentityWriter.scala

```diff
@@ -12,18 +12,18 @@ import org.http4s.util.chunk._
 import org.log4s.getLogger
 import scala.concurrent.{ExecutionContext, Future}
 
-private[http4s] class IdentityWriter[F[_]](size: Long, out: TailStage[ByteBuffer])(
+private[http4s] class IdentityWriter[F[_]](size: Int, out: TailStage[ByteBuffer])(
```

Comment:
What forced this back to `Int`?

Reply:
`take` on `Chunk` is Int, while on `Segment` it's Long. I think before it it used the inherited take from Segment.